src/lib/tvgPictureImpl.h

```diff
@@ -89,7 +89,7 @@ struct Picture::Impl
         return ret;
     }
 
-    uint32_t reload()
+    uint32_t reload(int32_t colorSpace)
```

Comment:
Suggest setting the default value here.
uint32_t reload(uint32_t colorSpace = -1)